website-next/pages/projects.tsx

```diff
@@ -0,0 +1,6 @@
+import * as React from 'react'
+import { ProjectsPage } from '../components/projects/projects-page-component'
+
+export default function Projects() {
+  return <ProjectsPage />
```

Comment:
to keep things readable, I copied the old projects page into `components/` and only reference it here. I think this is a good practice going forward, try to keep the root files clean so we can use them mainly for routing